src/resolvers/admin.ts

```diff
@@ -23,4 +24,20 @@ export class AdminResolver {
     await killSwitch.save()
     return { isKilled }
   }
+
+  @Mutation(() => GeneratedSignature)
+  async generateSignature(@Arg('bytes', () => String) bytes: string) {
+    if (!config.appPrivateKey) {
+      throw new Error('No PRIVATE_KEY provided to generate signature')
+    }
+
+    if (!cryptoIsReady()) {
+      throw new Error('@polkadot/util-crypto is not ready')
+    }
+
+    const keypair = sr25519PairFromSeed(config.appPrivateKey)
+    const key = sr25519Sign(bytes, keypair)
+
+    return { signature: base64Encode(key) }
```

Comment:
Any specific reason you picked base64 encoding? Maybe hex would be a better choice as that's most often used scheme for signatures, as far as I'm aware